sympy/vector/coordsysrect.py

```diff
@@ -10,7 +10,16 @@
                                     SpaceOrienter, QuaternionOrienter)
 
 
-class CoordSysCartesian(Basic):
+def CoordSysCartesian(*args, **kwargs):
+    SymPyDeprecationWarning(
```

Comment:
> Also please add it to https://github.com/sympy/sympy/wiki/Release-Notes-for-1.1 (and add any other changes you know of for this release to that document).

I added information to release notes for every changes which I made in `vector` module.

> It needs to be specified as the issue argument. This is a required argument now in the 1.1 branch.

I don't understand. Could you explain what means "issue argument"?